Loading gpytorch/gpu-cuda-11.6/pytorch-1.11.0/1.8.0
  Loading requirement: cudnn/8.4.1 pytorch/gpu-cuda-11.6/1.11.0
ERROR: Could not find a version that satisfies the requirement scipy.stats (from versions: none)
ERROR: No matching distribution found for scipy.stats
ERROR: Could not find a version that satisfies the requirement scipy.interpolate (from versions: none)
ERROR: No matching distribution found for scipy.interpolate
/mpcdf/soft/SLE_15/packages/skylake/pytorch/gpu-cuda-11.6/anaconda_3_2021.11-2021.11/1.11.0/lib/python3.9/site-packages/torch/functional.py:568: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  /home/abuild/rpmbuild/BUILD/pytorch-1.11.0/aten/src/ATen/native/TensorShape.cpp:2227.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
Traceback (most recent call last):
  File "/u/dvoss/al_pmssmwithgp/model/main.py", line 196, in <module>
    gp_pipeline.run_for_iterations(start_iter, end_iter)
  File "/u/dvoss/al_pmssmwithgp/model/main.py", line 92, in run_for_iterations
    data_loader.load_initial_data()
  File "/raven/u/dvoss/al_pmssmwithgp/model/data_loader.py", line 62, in load_initial_data
    self.x_valid = torch.stack([torch.tensor(limited_df['IN_M_1'].values[-self.valid_points:], dtype=torch.float32), torch.tensor(limited_df['IN_M_2'].values[self.valid_points:], dtype=torch.float32)], dim=1).to(self.device)
RuntimeError: stack expects each tensor to be equal size, but got [300] at entry 0 and [9698] at entry 1
srun: error: ravg1078: task 0: Exited with exit code 1
srun: Terminating StepId=13883837.0
